tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+
+# RabbitMQ C AMQP tutorials
+
+## Introduction
+
+This directory holds example tutorial code that implement the six tutorials 
+found at: 
+
+ - <https://www.rabbitmq.com/getstarted.html>
+
+#### Prereqs:
+- PHP5.6.x
+- librabbitmq 0.5.x
+- php-amqp pecl extension 1.4.0
+
+In order to actually execute these scripts you will need a RabbitMQ server.
+
+The easiest way (and the method used to test these as they were created) is to use
```

Comment:
I would avoid proclaiming that it is the easiest way. On OS X and Windows installing docker is not the same as on Linux. Apart that, we already ship Vagrant environment with provisioning script. See details in [readme file  under Development section](https://github.com/pdezwart/php-amqp#development).